### PR TITLE
Renamed test.md to DOCKER.md 

### PR DESCRIPTION
**Pull request by Adesoji1**
@mostafaelhoushi  Renamed test.md to DOCKER.md and referenced it in README.md for consistency

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +149,10 @@ $ torchrun correctness.py --model facebook/layerskip-llama2-7B \
     --output_dir ./logs
 ```
 
+## Using Docker
+
+Kindy check [Docker.md](DOCKER.md) to setup the project using docker
```
Comment (mostafaelhoushi) [inline README.md:154]:
```suggestion
Kindy check [DOCKER.md](DOCKER.md) to setup the project using docker
```

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# entrypoint.sh
```
Comment (mostafaelhoushi) [inline entrypoint.sh:2]: To meet the legal requirements, is it possible to add this copyright header?

```suggestion

# Copyright (c) Meta Platforms, Inc. and affiliates.

# This source code is licensed under the license found in the
# LICENSE file in the root directory of this source tree.

# entrypoint.sh
```

**Review by mostafaelhoushi — COMMENTED** [review]
Thanks @Adesoji1 !
I just made two minor comments. Can you please take a look at them.

**Comment by Adesoji1** [comment]
@mostafaelhoushi  i have done all you require, kindly vet them. Thank you once again for your support.

**Review by mostafaelhoushi — APPROVED** [review]
Thanks @Adesoji1 for addressing the changes. Highly appreciate that!
Merging the PR.

**Pull request merged**